### DISCUSSION TITLE
It can't connect the postgresql by deployment with docker-compose for 2.0.0?

### DISCUSSION THREAD

**Discussion by harryas** [apache/dolphinscheduler#7467] [Q&A] [answered]
I follow the [QuickStart in Docker ](https://dolphinscheduler.apache.org/zh-cn/docs/latest/user_doc/guide/installation/docker.html) to deploy the 2.0.0. But it always reported the same error. What things can I do to resolve it?
![image](https://user-images.githubusercontent.com/24936375/146507015-8d83b748-a0e4-429d-a324-8d12e06bfa78.png)

I have modified the docker-compose.yml to config the port 5432 for postgresql image, and connected it with navicat, also created the tables necessary.

**Comment by davidzollo** [comment, marked as answer]
yes, the docker version is not updated in time,  I think you can have a try tomorrow
